app.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const app = express();
+
+app.use(express.static('public'))
+ 
+app.get('/', (request, response, next) => {
+  response.sendFile(__dirname + '/views/index.html');
+});
+
```

Comment:
Esta forma está bien, pero mejor que utilicéis esta otra, ya que con render puedes pasar parametros :

app.get('/', (req, res, next) => res.render('index'))

También  res, req, next, es igual de legible y lo verás siempre así.